File: semestr-6/Algorytmy i struktury danych 2/lab-7/main.py


filename = 'Podzielnosc_0001.csv'


with open(filename, 'r') as f:
    data = f.read()
    f.close()

data = [int(x.strip()) for x in data.splitlines() if x]

data = data[1:]

n = 5

data2 = [x%5 == 0 for x in data]

data3 = []

for x in data2:
    if len(data3) ==0:
        data3.append(x)
    else:
        data

File: semestr-6/Algorytmy i struktury danych 2/lab-7/main.go
package main

import (
	"encoding/csv"
	"fmt"
	"os"
	"strconv"
)

func main() {
	csvFile, err := os.Open("Podzielnosc_0001.csv")
	if err != nil {
		fmt.Println(err)
	}
	defer csvFile.Close()

	csvLines, err := csv.NewReader(csvFile).ReadAll()
	if err != nil {
		fmt.Println(err)
	}

	var array []int

	for _, line := range csvLines {
		var x, _ = strconv.Atoi(line[0])
		array = append(array, x)
	}

	array = pop

	fmt.Println(array)
	var n int
	fmt.Printf("Proszę podać liczbę n: ")
	fmt.Scanf("%d", &n)
	fmt.Println(n)

}
